doc/authorization.md

```diff
@@ -0,0 +1,165 @@
+```
+
+(access-management)=
+### Access management
+
+In LXD, identities cannot be granted permissions directly. Instead, permissions are added to groups, and groups are added to identities.
+To create a group, run:
+
+    lxc auth group create <group_name>
+
+To add a group to an identity, run:
+
+    lxc auth identity group add <authentication_method>/<identifier> <group_name>
+
+For OIDC clients, this is:
+
+    lxc auth identity group add oidc/<email_address> <group_name>
+
+The identity is now a member of the group. To add permissions to the group, run:
+
+    lxc auth group permission add <group_name> <entity_type> [<entity_name>] <entitlement> [<key>=<value>...]
+
+Here are some examples:
+
+- `lxc auth group permission add administrator server admin` grants members of `administrator` the `admin` entitlement on `server`.
+- `lxc auth group permission add junior-dev project sandbox operator` grants members of `junior-dev` the `operator` entitlement on project `sandbox`.
+- `lxc auth group permission add my-group instance c1 user project=default` grants members of `my-group` the `user` entitlement on instance `c1` in project `default`.
+
+Some entity types require more than one supplementary argument to uniquely specify the entity.
+For example, entities of type `storage_volume` and `storage_bucket` require an additional `pool=<storage_pool_name>` argument.
+
+(identity-provider-groups)=
+### Identity provider groups
+
+It is common practice to manage users, roles, and groups centrally via an identity provider (IdP).
+In LXD, identity provider groups allow groups that are defined by the IdP to be mapped to LXD groups.
+When an OIDC client makes a request to LXD, any groups that can be extracted from the client's identity token are mapped to LXD groups, giving the client the same effective permissions.
+
+To configure IdP group mappings in LXD, first you must configure your IdP to add groups to identity and access tokens as a custom claim.
+This configuration depends on your IdP.
+In [Auth0](https://auth0.com/), you can add the "roles" that a user has as a custom claim via an [action](https://community.auth0.com/t/how-to-add-roles-and-permissions-to-the-id-token-using-actions/84506).
+Alternatively, if {abbr}`RBAC (role-based access control)` is enabled for the audience, a "permissions" claim can be added automatically.
+In Keycloak, you can define a [mapper](https://keycloak.discourse.group/t/anyway-to-include-user-groups-into-my-jwt-token/8715) to set Keycloak groups in the token.
+
+LXD must then be configured to extract this claim.
+To do so, set the value of the [`oidc.groups.claim`](server-options-oidc) configuration key to the value of the field name of the custom claim:
+
+    lxc config set oidc.groups.claim=<claim_name>
+
+LXD will then expect the claim to be set in identity and access tokens as a JSON array containing a string value for each group name.
```

Comment:
Not sure I understand ... maybe:

```suggestion
LXD will then expect the claim to to contain the group names (the identity and access token should contain a JSON array containing a string value for each group name).
```

Reply:
How about:

> LXD will then expect a claim to present in identity and access tokens with this name. The value of the claim must be a JSON array containing a string value for each IdP group name.

Reply:
The part I don't understand is "a claim to present in identity and access tokens". Might just be me not understanding what a claim is. :wink:

Maybe: "LXD will then expect the identity and access tokens to contain a claim with this name."